web/src/components/TaskProgressCheckbox.tsx

```diff
@@ -94,7 +94,7 @@ export const TaskProgressCheckbox = (props: Props): ReactElement => {
     updateQueue
       .update()
       .then(() => {
-        setSuccessSnackbarIsOpen(true);
+        if (!(isFormationTask(props.taskId) && nextStatus === "COMPLETED")) setSuccessSnackbarIsOpen(true);
```

Comment:
So we need this because when formationCheckbox is set to complete we re-direct to the dashboad and the dashboard has its own code to make the same snackbar appear. So this would just flash the snackbar before the page redirect

Also it would queue the text in the screen reader and the snackbar would be read twice.

Reply:
Should we add a test to ensure this behavior?

Reply:
I believe that's what the added test is testing for, unless I'm reading it wrong. It's testing to make sure that the snackbar still appears on the dashboard if you are coming from the formation task. 

Unless you're saying a test to make sure it doesn't appear twice?

Reply:
No JP is right, so we have a test to test for what you described @SirSamuelJoseph but we don't have a test to make sure we don't display the snackbar on the task page before you move to the dashboard.